components/landing/topCollections/TopCollectionsCard/TopCollectionsCard.scss

```diff
@@ -0,0 +1,37 @@
+.top-collection-card,
```

Comment:
Create a shared component instead of shared scss files?

Reply:
done pls check

Reply:
prefer something like this, then we can inline tailwind classes

```vue
<template>
  <div
    class="top-collection-card w-full border border-card-border-color hover:border-border-color bg-background-color group">
    <nuxt-link
      class="flex flex-col hover:text-text-color"
      rel="nofollow noopener noreferrer"
      :to="`${collection?.id && `/${urlPrefix}/collection/${collection.id}`}`">
      <div
        class="top-collection-card-banner group-hover:opacity-[0.85] bg-center bg-cover"
        :style="{ backgroundImage: `url(${collection?.image})` }"
        :class="`${!collection && 'bg-k-shade animate-pulse'}`">
        <div class="top-collection-card-banner-avatar-container">
          <BasicImage
            custom-class="top-collection-card-banner-avatar-inner [&>img]:border [&>img]:border-card-border-color !p-0"
            :src="collection?.image || placeholder" />
        </div>
      </div>

      <div class="top-collection-card-info border-t border-card-border-color">
        <p
          class="collection-name font-bold text-base whitespace-nowrap text-center truncate">
          <span v-if="collection?.name">{{ collection.name }}</span>
          <span v-else class="inline-block">
            <NeoSkeleton no-margin :rounded="false" width="160" height="24" />
          </span>
        </p>

        <div class="info-fields">
          <div>
            <p class="capitalize text-k-grey text-xs">
              <span v-if="collection?.floorPrice || collection?.floor">
                {{ $t('price') }}
              </span>
              <NeoSkeleton
                v-else
                no-margin
                :rounded="false"
                width="40"
                height="12" />
            </p>

            <div
              v-if="collection?.floorPrice || collection?.floor"
              class="flex gap-2 items-center max-md:flex-row-reverse">
              <CommonTokenMoney
                :value="collection.floorPrice || collection.floor"
                inline
                :round="2" />

              <div v-if="formattedDiffPercent" :class="color" class="text-xs">
                {{ formattedDiffPercent }}
              </div>
            </div>
            <NeoSkeleton
              v-else
              no-margin
              :rounded="false"
              width="80"
              height="20"
              class="mt-1" />
          </div>

          <div>
            <p class="capitalize text-k-grey text-end text-xs">
              <span v-if="collection">{{ $t('volume') }}</span>
              <NeoSkeleton
                v-else
                no-margin
                :rounded="false"
                width="40"
                height="12" />
            </p>
            <div class="flex gap-2 items-center">
              <CommonTokenMoney
                v-if="collection"
                :value="volume"
                inline
                :round="1" />
              <NeoSkeleton
                v-else
                no-margin
                :rounded="false"
                width="80"
                height="20" />
            </div>
          </div>
        </div>
      </div>
    </nuxt-link>
  </div>
</template>
```

Reply:
that one is `TopCollectionsCard` component

Reply:
fixed